pyproject.toml

```diff
@@ -27,7 +27,8 @@ dependencies = [
     "numpy",
     "matplotlib",
     "networkx",
-    "ndcube"
+    "ndcube",
+    "sunkit_image"
```

Comment:
This need to go in the `requirements.txt` for punch-mission too. Please open a PR there.